### ISSUE TITLE
Atmos suits do not protect against flamethrowers

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#12525]
(WEB REPORT BY: streptomutans REMOTE: 172.93.109.202:7777)
When someone fires a flamethrower at you while you wear an atmos suit, one hit crits you and the second ends it all.
I do not think it is quite plausible for the flamethrower to have such an effect on atmos suits since they can also survive plasma fires.

**Comment by ghost** [comment]
This is intended as they are for the flame throwers in the tomb of Rafid away mission

While your thing makes sense, you would need to modify the tomb of rafid flamethrower traps so that they can still throw 'cursed fire'

**Comment by Shadowmech88** [comment]
It's not a bug, this is definitely intended even for regular flamethrowers.

[Feature Request] would be a more appropriate tag.

**Comment by Ephx1** [comment]
Why would it be intended?
Who said, "Yes, this suit that can withstand temperatures nearing the sun does not need to be protective against my flamethrower update"?
I want to slap that person.

**Comment by Exxion** [comment]
Yeah that's turbo dumb

**Comment by Shadowmech88** [comment]
Well, atmos suits can't withstand the flame from a welding tool either, can they?

**Comment by Exxion** [comment]
The flame of a welding torch is almost 3500 degrees Celsius and specifically designed for melting metal
Whereas a flamethrower is not

But you're right, it would make sense for a welding torch to not damage (as much, at least) through an atmos suit, but we don't currently have damage type-specific resistances

**Comment by Shadowmech88** [comment]
> we don't currently have damage type-specific resistances

Then we should add that. The flamethrower deals burn damage via the same proc that stuff like welders and lasers do, so we should just add general burn damage resist to atmos hardsuits rather than exempting them from flamethrowers specifically.

**Comment by Exxion** [comment]
Yeah, fair enough

**Comment by unid15** [comment]
I don't think this is intended; the flamethrowers in the away mission were supposed to be the same as normal flamethrowers

**Comment by Intigracy** [comment]
flammenwerfer

it werfs flammen

**Comment by FudgePucker** [comment]
Atmos hard suits can survive tempatures as hot as the surface of the sun. Don't do shit against flame throwers.
